README.md

```diff
@@ -145,6 +145,41 @@ bellhop.fetch('function', function(result) {
 });
 ```
 
+Futhermore, the structure allows data to be passed to respond() as:
+* data 
```

Comment:
Instead of this faux-TypeScript syntax that I suggested, let's just use plain old words:

> Furthermore, `respond()` accepts a plain object, string,  or number. If a function is passed, it will be called and the function's _return-value_ sent. If a promise is passed in or returned from a function that was passed in, that promise will be `await`-ed before it's value returned.